Admin/PageAdmin.php

```diff
@@ -327,15 +327,15 @@ protected function configureFormFields(FormMapper $formMapper)
         if (!$this->getSubject() || (!$this->getSubject()->isInternal() && !$this->getSubject()->isError())) {
             $formMapper
                 ->with('form_page.group_main_label')
-                    ->add('url', TextType::class, ['attr' => ['readonly' => 'readonly']])
+                    ->add('url', TextType::class, ['attr' => ['readonly' => true]])
```

Comment:
Wasn't that valid?

Reply:
I was checking that, it seems like it is not valid, but it was casted correctly: https://github.com/symfony/symfony/pull/10676/files#diff-22a03074077455f3d1128486ef5d9851R82

Using a boolean seems to be favored as this test shows: https://github.com/symfony/symfony/pull/10676/files#diff-e11656489f1e01413320bd5edff0f0c7R1917

Reply:
Also it's a flag, which should be able to be turned on or off -> boolean